### PR TITLE
apiv1(ticdc) :  changefeed API return creator_version

### LINKED ISSUE #8728: cli(ticdc):  cli query command not show creator_version
see https://asktug.com/t/topic/1004117?replies_to_post_number=2

### PR DESCRIPTION
**Pull request by sdojjy**
### What problem does this PR solve?

Issue Number: close #8728

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
fix a bug that cli query changefeed command not show changefeed creator version
```

### CODE REVIEW

**Review by asddongmen — APPROVED** [review]

File: cdc/api/v1/api.go [hunk 1]
```diff
@@ -256,6 +256,7 @@ func (h *OpenAPI) GetChangefeed(c *gin.Context) {
 		Engine:         info.Engine,
 		FeedState:      info.State,
 		TaskStatus:     taskStatus,
+		CreatorVersion: info.CreatorVersion,
```
Comment (nongfushanquan) [inline cdc/api/v1/api.go:259]: add a test case?

Comment (sdojjy) [reply cdc/api/v1/api.go:259]: done

**Pull request merged**